test/reports/bootstrap-voting/rest-gateway-0-rest.json

```diff
@@ -4,6 +4,9 @@
     "description": "catapult development network"
   },
   "port": 3000,
+  "protocol": "HTTP",
```

Comment:
HTTP? Should it be HTTPS?

Reply:
In the `shared.yml` file the protocol(restProtocol) is defaulted to HTTP and only if the HTTPS selection is `Native` on `wizard` command or manually changed to HTTPS in the custom-preset then it will be HTTPS.  But this should be the user's conscious decision I think.